Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'stanford-mods', '~> 2.6'
 gem 'lockfile'       # file locks needed for mutual exclusion during rollup and index addition processes
 gem 'mini_exiftool'  # was-seed-preassembly thumbnail creation
 gem 'mini_magick'    # was-seed-preassembly thumbnail creation
-gem 'assembly-image' # was-seed-preassembly thumbnail creation
+gem 'assembly-image', '~> 2.0' # was-seed-preassembly thumbnail creation; 2.0.0 uses ruby-vips for better jp2 creation
```

Comment:
I think this is a little bit confusing because vips makes tiffs.  kakadu makes jp2s from the tiffs.

Reply:
I'll change the comment.